WinImageBuilder.psm1

```diff
@@ -361,14 +361,25 @@ function Download-CloudbaseInit {
         [Parameter(Mandatory=$true)]
         [string]$resourcesDir,
         [Parameter(Mandatory=$true)]

```

Comment:
The commit title message: Parameterize Cloudbase-Init installer to use Stable or Beta one